icu4c/source/test/intltest/units_test.cpp

```diff
@@ -489,6 +489,22 @@ void UnitsTest::testComplexUnitsConverter() {
           Measure(0, MeasureUnit::createMeter(status), status)},
          2,
          0},
+        {"Mixed Units test 1",
+         "meter",
+         "foot-and-inch",
+         3.0,
+         {Measure(9, MeasureUnit::createFoot(status), status),
+          Measure(10.110236220472423, MeasureUnit::createInch(status), status)},
+         2,
+         0},
+        {"Mixed Units test 1",
```

Comment:
Give it a different name? This is to help finding the unit test that fails. You could also call them "Mixed Units: foot-and-inch" and "Mixed Units: inch-and-foot".

In C++ I sometimes add test names primarily because it's too much of a pain to construct descriptive strings from other test case fields at runtime. ;-)

Reply:
good idea